test/functional/mempool_ephemeral_dust.py

```diff
@@ -325,7 +325,7 @@ def test_reorgs(self):
         dusty_tx, _ = self.create_ephemeral_dust_package(tx_version=3)
         assert_raises_rpc_error(-26, "min relay fee not met", self.nodes[0].sendrawtransaction, dusty_tx["hex"])
 
-        block_res = self.nodes[0].rpc.generateblock(self.wallet.get_address(), [dusty_tx["hex"]])
+        block_res = self.generateblock(self.nodes[0], self.wallet.get_address(), [dusty_tx["hex"]], sync_fun=self.no_op)
```

Comment:
```
sync_fun=self.no_op
```

Note for myself and other reviewers: Syncing is not done in this test because the nodes are disconnected on line 321 and hence, syncing all the nodes would result in assertion errors.

```
  File "/Users/rkrux/projects/bitcoin/test/functional/test_framework/test_framework.py", line 769, in sync_blocks
    assert (all([len(x.getpeerinfo()) for x in rpc_connections]))
            ~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError
```